specific/cmo_account/models/account.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is account.invoice model, so please change file name to **account_invoice.py**